bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/i18n/ferroamp.properties

```diff
@@ -0,0 +1,3 @@
+# FIXME: please add all English translations to this file so the texts can be translated using Crowdin
```

Comment:
One of the last steps after the review changes are checked in, the i18n plugin should be executed:
https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file